web/styles/compose.css

```diff
@@ -1236,7 +1236,7 @@ textarea.new_message_textarea,
         transform: scale(0.96);
     }
 
-    &:focus {
+    &:focus:not(:focus-visible) {
```

Comment:
Two issues here:

1. `:focus` is a separate pseudoclass from `:focus-visible`, so I don't understand what's gained by `:not()` here.
2. The issue #27117 makes clear that `:focus` should take browser defaults, so if anything, this and other remaining `:focus` styles should either be removed or potentially take [a value of `revert`](https://developer.mozilla.org/en-US/docs/Web/CSS/revert)